pkg/new-ui/v1beta1/frontend/src/assets/js/d3.parcoords.js

```diff
@@ -0,0 +1,2867 @@
+d3.parcoords = function(config) {
```

Comment:
We can't just install d3 in node modules ?

Reply:
We've installed d3 through node from the other discussions, so I'll resolve this